controller/src/controller/services/fileserver.py

```diff
@@ -228,16 +254,16 @@ async def reconcile(self) -> None:
         seen_users = {u for u in seen if u not in to_delete}
         for username in (known_users - seen_users) | to_delete:
             if username in self._servers:
-                last_modified = self._servers[username].last_modified
-                if last_modified > start:
+                if self._servers[username].modified_since(start):
                     continue
             if username in to_delete:
                 msg = "Removing broken fileserver for user"
                 self._logger.warning(msg, user=username)
             else:
                 msg = "No file server job for user, removing remnants"
                 self._logger.warning(msg, user=username)
-            await self.delete(username)
+            with contextlib.suppress(UnknownUserError):
```

Comment:
I don't think I understand in what case delete(username) could throw the UnknownUserError.  If there's some way multiple things can be working on the user at once, do we want some explicit locking mechanism?